tests/benchmarks/igraph_rewire.c

```diff
@@ -17,6 +17,7 @@
 */
 
 #include <igraph.h>
+#include "igraph_constants.h"
```

Comment:
In benchmarks and tests, let's stick to `<igraph.h>` only, with angle brackets. Other headers should be considered internal.